op-chain-ops/cmd/op-migrate/main.go

```diff
@@ -1,312 +0,0 @@
-package main
```

Comment:
I'd leave this untouched and just add a celo-migrate. It'll help prevent conflicts

Reply:
Oh right 👍  The name (op-migrate) confused me